core/templates/pages/collection-editor-page/services/collection-editor-state.service.spec.ts

```diff
@@ -318,15 +318,6 @@ describe('Collection editor state service', function() {
     expect(collection.getCollectionNodes()).toEqual([]);
   });
 
-  it('should initially return an empty collection rights', function() {
```

Comment:
Why delete this?

Reply:
The _collectionRight is no more initialized so it will return undefined. Thus this test is redundant now